integration/container/kill_test.go

```diff
@@ -21,11 +21,13 @@ func TestKillContainerInvalidSignal(t *testing.T) {
 	id := container.Run(ctx, t, client)
 
 	err := client.ContainerKill(ctx, id, "0")
-	assert.Error(t, err, "Error response from daemon: Invalid signal: 0")
+	assert.ErrorContains(t, err, "Error response from daemon:")
+	assert.ErrorContains(t, err, "nvalid signal: 0")
```

Comment:
Should this be "nvalid" or "invalid"?

Reply:
ah, so this was a little hack; the update has some linting fixes, one of them is "errors should not be capitalised or contain interpunction" so it changed from `Invalid signal` to `invalid signal`. Tried to keep it "portable" without doing a `string.ToLower()`. Probably not strictly needed (in case we wanted to run the integration tests against different daemon versions, but I guess we won't be doing that)

Reply:
That'd be worth a comment if you're inclined to make a minor fix here.

Reply:
I rebased, and added a comment 👍